tests/integration/session/test_sign_out_and_exit.py

```diff
@@ -72,17 +70,31 @@ def _launch_and_submit_questionnaire(self, schema, **kwargs):
         self.post()
         self.assertInUrl("/thank-you")
 
-    def test_no_account_service_log_out_url_redirects_to_signed_out_page(self):
+    def test_redirects_to_account_service_log_out_url_using_base_url_from_claims(self):
         self._launch_and_submit_questionnaire(
-            schema="test_new_hub_section_required_and_enabled"
+            schema="test_new_hub_section_required_and_enabled",
+            account_service_url=ACCOUNT_SERVICE_BASE_URL,
         )
-        self.get(SIGN_OUT_URL_PATH, follow_redirects=False)
-        self.assertInUrl(SIGN_OUT_URL_PATH)
+        self.signOut()
+        self.assertInRedirect(ACCOUNT_SERVICE_LOG_OUT_URL)
 
-    def test_redirects_to_account_service_log_out_url_when_present(self):
-        self._launch_and_submit_questionnaire(
-            schema="test_new_hub_section_required_and_enabled",
-            account_service_log_out_url=ACCOUNT_SERVICE_LOG_OUT_URL,
+
+class TestCensusSignOut(IntegrationTestCase):
+    def setUp(self):
+        # Perform setup steps
```

Comment:
I hope everyone inside a setup knows there are steps 😆 . I don't think we need the comment

Reply:
Copy and pasted existing test 🤷